content/applications/general/voip/onsip.rst

```diff
@@ -5,94 +5,166 @@ Use VoIP services in Odoo with OnSIP
+
+After opening an OnSIP account, follow the configuration procedure below to configure it on an Odoo
+database.
 
 Configuration
 =============
 
-Go to Apps and install the module **VoIP OnSIP**.
+To configure the Odoo database to connect to OnSIP services, first navigate to the
+:menuselection:`Apps application` from the main Odoo dashboard. Then, remove the default `Apps`
+filter from the :guilabel:`Search...` bar, and search for `VoIP OnSIP`.
+
+Next, install the :guilabel:`VOIP OnSIP` module.
 
-.. image:: onsip/onsip01.png
+.. image:: onsip/install-onsip.png
    :align: center
+   :alt: View of OnSIP app in the app search results.
 
-Go to Settings/General Settings. In the section Integrations/Asterisk (VoIP), fill in the 3 fields:
+Odoo VoIP setting
+-----------------
 
-- **OnSIP Domain** is the domain you chose when creating an account on www.onsip.com. If you don't
-  know it, log in to https://admin.onsip.com/ and you will see it in the top right corner of the
-  screen.
-- **WebSocket** should contain wss://edge.sip.onsip.com
-- **Mode** should be Production
+After installing the *VOIP OnSIP* module, go to the :menuselection:`Settings app`, scroll down to
+the :guilabel:`Integrations` section, and locate the :guilabel:`Asterisk (VoIP)` fields. Then,
+proceed to fill in those three fields with the following information:
 
-.. image:: onsip/onsip02.png
+- :guilabel:`OnSIP Domain`: the domain that was assigned when creating an account on `OnSIP
+  <https://www.onsip.com/>`_.
+- :guilabel:`WebSocket`: `wss://edge.sip.onsip.com`
+- :guilabel:`VoIP Environment`: :guilabel:`Production`
+
+.. image:: onsip/asterisk-setting.png
    :align: center
+   :alt: VoIP configuration settings in Odoo Settings app.
+
+.. tip::
+   To access the OnSIP domain, navigate to `OnSIP <https://www.onsip.com/>`_ and log in. Then, click
+   the :guilabel:`Administrators` link in the top-right of the page.
+
+   Next, in the left menu, click :guilabel:`Users`, and then select any user. By default, the
+   selected user opens on the :guilabel:`User Info` tab.
+
+   Click on the :guilabel:`Phone Settings` tab to reveal OnSIP configuration credentials. In the
+   first column, last in the list of credentials, is :guilabel:`Domain`.
+
+   .. image:: onsip/domain-setting.png
+      :align: center
+      :alt: Domain setting revealed (highlighted) on administrative panel of OnSIP management
+            console.
+
+Odoo user setting
+-----------------
+
+Next, the user needs to be set up in Odoo. Every user associated with an OnSIP user **must** also be
+configured in the Odoo user's settings/preferences.
+
+To do that, navigate to :menuselection:`Settings app --> Manage Users --> Select the User`.
+
+On the user form, click :guilabel:`Edit` to configure the user's OnSIP account. Then, click the
+:guilabel:`Preferences` tab, and scroll to the :guilabel:`VoIP Configuration` section.
+
+In this section, fill in the fields with OnSIP credentials.
+
+.. tip::
```

Comment:
this **tip** is a word-for-word match as the other one, minus the last sentence about the **Domain** credential.
think we can get rid of it?